engine/runtime-fat-jar/src/main/java/module-info.java

```diff
@@ -13,7 +13,8 @@
 
   provides com.oracle.truffle.api.provider.TruffleLanguageProvider with
       org.enso.interpreter.EnsoLanguageProvider,
-      org.enso.interpreter.epb.EpbLanguageProvider;
+      org.enso.interpreter.epb.EpbLanguageProvider,
+      org.enso.interpreter.arrow.ArrowLanguageProvider;
```

Comment:
I got my answer from Hubert: Because right now `ArrowLanguage` ends up in the `runtime` uberJAR. I'd be glad if that changed and `ArrowLanguage` had its own JAR.